RELEASE_NOTES.md

```diff
@@ -1,178 +1,277 @@
-in CSM. There is no longer a separate SAT product stream to install. SAT 2.6 releases,
-which accompanied CSM 1.5, are the last releases of SAT as a separate product. For more
-information, see [SAT in CSM](operations/system_admin_toolkit/about_sat/SAT_in_CSM.md).
 
 ### New hardware support
+TODO: Is this still relevant?
 
-* CSM 1.6.1 supports systems with NVIDIA CPUs and GPUs. CSM 1.6.0 does not support NVIDIA hardware.
-
-### New software support
-
-* Bump `iuf-cli` version to 1.6.15
-* Bump `cray-nls` version to 4.0.15
-* Added support to upgrade CSM through IUF
-* Added support for iSCSI based boot content projection for `rootfs` and `PE` images
+* *IMPORTANT** Systems with NVIDIA CPUs and GPUs must see [Known issues](#known-issues)
 
 ## Improvements
 
-### Automation improvements
-
-* CSM can now be upgraded with IUF.
+### General Improvements
+
+* CASMPET-7263 TESTS: goss-servers should delay start until hostname is set
+* CASMCMS-9188 TESTS: cmsdev: Add explicit check for blank CFS ID field
+* CASMPET-7262 TESTS: After installing csm-testing RPM, automatically restart goss-servers
+* CASMCMS-9189 Update CFS API spec to reject invalid component creation/update requests
+* CASMCMS-9197 Bypass needless work in some CFS queries
+* CASMCMS-9200 Make CFS Options class thread-safe and more performant
+* CASMCMS-9202 Add ability to create CFS source and specify secret name instead of username/pw
+* CASMCMS-9207 Update CFS API spec with actual status code for successful source restore
+* CASMCMS-9208 Decode source_name when restoring source
+* CASMCMS-9211 Improve CFS config delete performance on scale systems
+* CASMHMS-6299 PCS/TRS: Mitigate resource leaks / heavy usage
+* CASMPET-7288 TESTS: Minimize use of system Python packages in virtual environment
+* CASMPET-7290 TESTS: Add pylint to csm-testing build pipeline
+* MTL-2514 Pin cni / cni-plugins RPM's to the latest version from SLE-Module-Containers
+* CRAYSAT-1896 Update "sat bootprep" to support CFS v2 or v3
+* SSI-14310 Update CSM 1.6 configuration page to include USS product changes.
+* CRAYSAT-1923 Update "sat bootsys" to support CFS v2 or v3
+* CRAYSAT-1936 Add ability to sort reports by multiple fields
+* CASMINST-7039 Cleanup previous/old SquashFS images during upgrade
+* CRAYSAT-1942 Drop internal default values for rootfs_provider{,passthrough} from sat bootprep
+* CASMCMS-9225 Add context managers around BOS requests/sessions; enable paging of BOS components
+* CASMMON-463 Update customization.yaml for sma victoria metrics pvc size
+* CASMCMS-9068 Allow customization of ipxe debug options
+* CASMINST-6968 Replace `sed` calls with `yq` in upgrade_control_plane.sh
+* CASMCMS-9177 Make BOS migration pod more polite
+* CASMHMS-6360 Make cray-hms-rts version consistent in csm manifest
+* CASMINST-6968 Replace `sed` calls with `yq` in upgrade_control_plane.sh
 
 ### Base platform component upgrades
```

Comment:
Need to remove any items from this list that have not had version changes from 1.6.0 to 1.6.1

Reply:
We don't have version changes from 1.6.0 to 1.6.1 for this section, so we should remove this section entirely.

Reply:
Since these release notes will be read by users coming from 1.5.X I think it might be best to keep them as an aggregate.
If want a 1.6.1-exclusive release notes we should add those into the `upgrade/1.6.1` directory (or in this case since we haven't had more than 1, the `upgrade/` directory). For now I think it's okay as a large aggregate, but it does make it confusing for users coming from 1.6.0.